expansions.txt

```diff
@@ -197,6 +197,7 @@ Neck Protection Mandate
 Necromancers Playing MTG
 Necrotizing Pineapple Music
 Nectar of the Programming Masses
+Nectar Packed Marigolds
```

Comment:
I feel that we are the bees who are in quest of nectar, we find this Nectar Packed Marigolds (NPM) to feed our-self. @npm-robot